neurom/ezy/neuron.py

```diff
@@ -111,6 +111,7 @@ def __init__(self, filename, iterable_type=np.array):
         self._nrn = load_neuron(filename, set_tree_type)
         self.soma = self._nrn.soma
         self.neurites = self._nrn.neurites
+        self.filename = self._nrn.id
```

Comment:
Yes, this is literally the filename without the extension. If we want to keep the name and remove the path of the file I can rename this to name and keep only the name striping the path data. Is this better?